index.js

```diff
@@ -131,7 +141,7 @@ class KeyringController extends EventEmitter {
       throw new Error('KeyringController - First Account not found.');
     }
 
-    await this.persistAllKeyrings(password);
+    await this.persistAllKeyrings();
```

Comment:
Why does this need to be called twice? Looks like the initial call is redundant or there's not enough separation of concerns for addNewKeyring to work regardless.

Reply:
Heh, it's from the first ever commit:
<img width="1034" alt="FirstCommit" src="https://user-images.githubusercontent.com/46655/200400330-2cfccd3f-2878-4af9-ac12-98d19eefaf8c.png">